### PR TITLE
v8.3: bump the latest TiDB version to v8.3

### PR DESCRIPTION
**Pull request by hfxsd**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [x] v8.3 (TiDB 8.3 versions)
- [ ] v8.2 (TiDB 8.2 versions)
- [ ] v8.1 (TiDB 8.1 versions)
- [ ] v8.0 (TiDB 8.0 versions)
- [ ] v7.5 (TiDB 7.5 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)

### What is the related PR or file link(s)?

- This PR is translated from:https://github.com/pingcap/docs-cn/pull/18390
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: upgrade-tidb-using-tiup.md [hunk 1]
```diff
@@ -21,11 +21,11 @@ This document is targeted for the following upgrade paths:
 > 3. **DO NOT** upgrade a TiDB cluster when a DDL statement is being executed in the cluster (usually for the time-consuming DDL statements such as `ADD INDEX` and the column type changes). Before the upgrade, it is recommended to use the [`ADMIN SHOW DDL`](/sql-statements/sql-statement-admin-show-ddl.md) command to check whether the TiDB cluster has an ongoing DDL job. If the cluster has a DDL job, to upgrade the cluster, wait until the DDL execution is finished or use the [`ADMIN CANCEL DDL`](/sql-statements/sql-statement-admin-cancel-ddl.md) command to cancel the DDL job before you upgrade the cluster.
```
Comment (Oreoxmt) [inline upgrade-tidb-using-tiup.md:21]: Lines 11-15 require updates
```
- Upgrade from TiDB 4.0 versions to TiDB 8.2.
- Upgrade from TiDB 5.0-5.4 versions to TiDB 8.2.
- Upgrade from TiDB 6.0-6.6 to TiDB 8.2.
- Upgrade from TiDB 7.0-7.6 to TiDB 8.2.
- Upgrade from TiDB 8.0-8.1 to TiDB 8.2.
```

File: system-variables.md [hunk 2]
```diff
@@ -6354,7 +6354,7 @@ Internally, the TiDB parser transforms the `SET TRANSACTION ISOLATION LEVEL [REA
 - Scope: NONE
 - Applies to hint [SET_VAR](/optimizer-hints.md#set_varvar_namevar_value): No
 - Default value: `8.0.11-TiDB-`(tidb version)
-- This variable returns the MySQL version, followed by the TiDB version. For example '8.0.11-TiDB-v8.2.0'.
+- This variable returns the MySQL version, followed by the TiDB version. For example '8.0.11-TiDB-v8.3.0'.
```
Comment (Oreoxmt) [inline system-variables.md:6357]: `version` needs updating in https://github.com/pingcap/docs-cn/pull/18390

Comment (hfxsd) [reply system-variables.md:6357]: https://github.com/pingcap/docs-cn/pull/18390/files#diff-ec2cbb115cc58fe3afe2862eb5148732d323cbf7ae35dea97bd381a1f8d0e8a0

File: upgrade-tidb-using-tiup.md (hunk 1 above)
Comment (hfxsd) [reply upgrade-tidb-using-tiup.md:21]: Done

File: tidb-binlog/deploy-tidb-binlog.md [hunk 3]
```diff
@@ -342,7 +342,7 @@ The following part shows how to use Pump and Drainer based on the nodes above.
         # to the kafka address in zookeeper
         # zookeeper-addrs = "127.0.0.1:2181"
         # kafka-addrs = "127.0.0.1:9092"
-        # kafka-version = "0.8.2.0"
+        # kafka-version = "0.8.3.0"
```
Comment (Oreoxmt) [inline tidb-binlog/deploy-tidb-binlog.md:345]:
```suggestion
        # kafka-version = "0.8.2.0"
```

File: tiup/tiup-component-cluster-patch.md [hunk 4]
```diff
@@ -29,7 +29,7 @@ Before running the `tiup cluster patch` command, you need to pack the binary pac
 1. Determine the following variables:
 
     - `${component}`: the name of the component to be replaced (such as `tidb`, `tikv`, or `pd`).
-    - `${version}`: the version of the component (such as `v8.2.0` or `v7.5.2`).
+    - `${version}`: the version of the component (such as `v8.3.0` or `v7.5.2`).
```
Comment (Oreoxmt) [inline tiup/tiup-component-cluster-patch.md:32]:
```suggestion
    - `${version}`: the version of the component (such as `v8.3.0` or `v7.5.3`).
```

File: tiup/tiup-playground.md [hunk 5]
```diff
@@ -184,10 +184,10 @@ tiup playground scale-in --pid 86526
 
 ## Deploy PD microservices
 
-Starting from v8.2.0, [PD microservice mode](/pd-microservices.md) (experimental) can be deployed using TiUP. You can deploy the `tso` microservice and `scheduling` microservice for your cluster using TiUP Playground as follows:
+Starting from v8.3.0, [PD microservice mode](/pd-microservices.md) (experimental) can be deployed using TiUP. You can deploy the `tso` microservice and `scheduling` microservice for your cluster using TiUP Playground as follows:
```
Comment (Oreoxmt) [inline tiup/tiup-playground.md:187]:
```suggestion
Starting from v8.2.0, [PD microservice mode](/pd-microservices.md) (experimental) can be deployed using TiUP. You can deploy the `tso` microservice and `scheduling` microservice for your cluster using TiUP Playground as follows:
```

**Review by Oreoxmt — APPROVED** [review]

**Pull request merged**